app/frontend/javascript/components/cookie_banner.test.mjs

```diff
@@ -3,7 +3,28 @@ import { CookieBanner } from './cookie_banner.mjs'
 
 describe('CookieBanner', () => {
   let cookieBanner
-  const cookieBannerHTML = readFileSync('tests/fixtures/cookie_banner.html', 'utf8')
+  const cookieBannerHTML = `
```

Comment:
I think these were a bit tidier as fixtures - was there a reason for moving them into the test file?

Reply:
I removed them mostly because it was a bit weird to have a `/spec` and a `/test` folder but its weird having this stubbed out HTML when there is already a view with this markup and feels like a bit of code smell where the HTML could easily go out of sync.

Looking at the two js component tests, are they needed? I feel like this functionality has already been tested by the capybara feature tests. I think the Capybara features test more functionality than these unit tests. Eg. checking if Google tag manager is injected into the page or not.